extensions/amp-byside-content/0.1/amp-byside-content.js

```diff
@@ -122,32 +150,55 @@ export class AmpBysideContent extends AMP.BaseElement {
    * @override
    */
   preconnectCallback(onLayout) {
-    if (this.iframeSrc_) {
-      this.preconnect.url(this.iframeSrc_, onLayout);
+    if (this.origin_) {
+      this.preconnect.url(this.origin_, onLayout);
     }
   }
 
   /** @override */
   buildCallback() {
     this.webcareId_ = user().assert(
         this.element.getAttribute('data-webcare-id'),
-        'The data-webcare-id attribute is required for <' + TAG_ + '> %s',
-        this.element);
+        'The data-webcare-id attribute is required for <%s> %s',
+        TAG_,
+        this.element
+    );
 
     this.label_ = user().assert(
         this.element.getAttribute('data-label'),
-        'The data-label attribute is required for <' + TAG_ + '> %s',
-        this.element);
+        'The data-label attribute is required for <%s> %s',
+        TAG_,
+        this.element
+    );
 
     this.webcareZone_ = (this.element.getAttribute('data-webcare-zone') ||
-		DEFAULT_WEBCARE_ZONE_);
+        DEFAULT_WEBCARE_ZONE_);
     this.channel_ = (this.element.getAttribute('data-channel') || '');
     this.lang_ = (this.element.getAttribute('data-lang') || DEFAULT_LANG_);
     this.fid_ = (this.element.getAttribute('data-fid') || '');
 
+    this.isResizable_ = this.element.hasAttribute('resizable');
+    if (this.isResizable_) {
+      this.element.setAttribute('scrolling', 'no');
+    }
+
+    if (!this.element.hasAttribute('frameborder')) {
```

Comment:
ditto above, it is strange to expose iframe attributes on this component. why don't the defaults work?

Reply:
As above, we are now assuming defaults.